### PR TITLE
Tray Icon annex to Soundnote App with a simple menu item [Close]

### PR DESCRIPTION
**Pull request by ronaldoats**
Tray Icon annex to Soundnote App with a simple menu item [Close]

### CODE REVIEW

**Comment by wb9688** [comment]
English please?
Op 1 feb. 2016 23:12 schreef "Ronaldo Tinoco" <notifications@github.com>:

File: app/public/js/system/guiConfig.js [hunk 1]
```diff
@@ -3,6 +3,32 @@
...
+var tray = new gui.Tray({ title: 'Tray', icon: 'soundnode.png' });
+
+// Give it a menu
+var menu = new gui.Menu();
+
+var visible = false;
+// Controlar visibilidad al dar click sobre el icono
+tray.click = function(){
+  if(visible){
+    guiConfig.getGUI.hide();
```
Comment (Ajeey) [inline app/public/js/system/guiConfig.js]: Shorter version 
visible ? guiConfig.getGUI.hide() : guiConfig.getGUI.show();

Comment (ronaldoats) [inline app/public/js/system/guiConfig.js]: thanks, I was adding a new commit.

**Comment by ronaldoats** [comment]
Please ignore the second commit, I'm not very experienced with this git.
Commits on February 2, 2016

**Comment by weblancaster** [comment]
I don't think this is necessary.

**Pull request closed without merging**